README.md

```diff
@@ -85,21 +109,22 @@ To enable requiring files without specifying the extension, you must add a `reso
 module.exports = {
   entry: './main.js',
   output: {
-    filename: 'bundle.js'       
+    filename: 'bundle.js'
   },
   module: {
     loaders: [
       { test: /\.coffee$/, loader: 'coffee-loader' },
-      { test: /\.js$/, loader: 'babel-loader' }
+      { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader'}
```

Comment:
Yes, very important one.

You removed the white space before the closing brace.